static/js/breakpoint-handler.js

```diff
@@ -22,18 +24,18 @@ function getViewportDimensions() {
  *  enter: callback when breakpoint region entered,
  *  leave: callback when breakpoint region exited
  * }
- * 
+ *
  */
 
-BreakpointHandler = function (opts) {
-    opts || (opts = {});
+function BreakpointHandler(opts) {
```

Comment:
Cool. Maybe that's something you guys should explore in the JS standards group. I'd just like to have rules I can follow on when to use either style, whatever those rules are.